src/engine/server/sv_init.cpp

```diff
@@ -437,6 +440,12 @@ void SV_SpawnServer(std::string pakname, std::string mapname)
 	SV_ShutdownGameProgs();
 
 	PrintBanner( "Server Initialization" )
+
+	if ( SV_Private( ServerPrivate::Public ) && cvar_abuse.Get().empty() )
```

Comment:
The condition is inverted here. Also I suggest including sv_networkScope so that it matches the condition for advertising to the master server. So `!SV_Private(ServerPrivate::NoAdvertise) && sv_networkScope >= 2`

Reply:
This is still not fixed

Reply:
Done.